README.md

```diff
@@ -3,148 +3,193 @@
+If you also want to run Plankton's unit tests, you may also install the development dependencies:
 
 ```
-$ export EPICS_CA_ADDR_LIST=172.17.255.255
+$ pip install -r requirements-dev.txt
 ```
 
-If the Docker network is configured differently, this IP needs to be changed accordingly.
+If you want to use the EPICS adapter, you will also need to configure EPICS environment variables correctly. If you only want to communicate using EPICS locally via the loopback device, you can configure it like this:
+
+```
+$ export EPICS_CA_AUTO_ADDR_LIST=NO
+$ export EPICS_CA_ADDR_LIST=localhost
+$ export EPICS_CAS_INTF_ADDR_LIST=localhost
+```
 
-## Interacting with the simulated chopper
+Once all dependencies and requirements are satisfied, Plankon can be run using the following general format (from inside the Plaknton dirrectory:
 
-The simulated chopper is exposed via a set of EPICS PVs. For a detailed description of those, see the table of available PVs below.
+```
+$ python simulation.py [plankton args] [-- [adapter args]]
+```
 
-Observe some available EPICS PVs in an automatically updating screen:
+You can then run Plankton as follows (from within the plankton directory):
 
 ```
-$ watch -n 1 caget SIM:State SIM:CmdL SIM:Spd-RB SIM:Spd SIM:Phs-RB SIM:Phs SIM:ParkAng-RB SIM:ParkAng
+$ python simulation.py -d chopper -p epics
 ```
 
-The following series of `caput`-commands, executed from a different terminal, will move the chopper to the specified
-speed and phase:
+Details about parameters for the various adapters, and differences between OSes are covered in the "Adapter Specifics" sections.
+
+
+## EPICS Adapter Specifics
+
+The EPICS adapter takes only one optional argument:
+
+- `-p` / `--prefix`: This string is prefixed to all PV names. Defaults to empty / no prefix.
+
+Arguments meant for the adapter should be separated from general Plankton arguments by a free-standing `--`. For example:
 
 ```
-$ caput SIM:CmdS init
-$ caput SIM:Spd 100.0
-$ caput SIM:Phs 23.0
-$ caput SIM:CmdS start
+$ docker run -itd dmscid/plankton -d chopper -p epics -- -p SIM1:
+$ python simulation.py -d chopper -p epics -- --prefix SIM2:
 ```
 
-It may take a while until the simulation reaches the `phase_locked` state.
+When using the EPICS adapter within a docker container, the PV will be served on the docker0 network (172.17.0.0/16).
+
+On Linux, this means that `EPICS_CA_ADDR_LIST` must include this networks broadcast address:
+
+```
+$ export EPICS_CA_AUTO_ADDR_LIST=NO
+$ export EPICS_CA_ADDR_LIST=172.17.255.255
+$ export EPICS_CAS_INTF_ADDR_LIST=localhost
+``` 
 
+On Windows and OSX, the docker0 network is inside of a virtual machine. If we want to communicate with it, we need to use an EPICS Gateway to forward EPICS requests and responses for us. We provide an [EPICS Gateway Docker image](https://hub.docker.com/r/dmscid/epics-gateway/) that can be used to do this relatively easily (detailed instructions on the linked page). 
```

Comment:
Yeah, that doesn't sound good. I think I'll reword to avoid "we" or "you" altogether.

The "We provide" can probably stay though.